solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -267,10 +319,18 @@
         "location": "[resourceGroup().location]",
         "documentDBApiVersion": "2016-03-19",
         "documentDBResourceId": "[resourceId('Microsoft.DocumentDb/databaseAccounts', parameters('documentDBName'))]",
+        "eventHubApiVersion": "2017-04-01",
+        "eventHubKeyName": "RootManageSharedAccessKey",
+        "eventHubKeyResource": "[resourceId('Microsoft.EventHub/namespaces/AuthorizationRules', parameters('eventHubNamespaceName'), variables('eventHubKeyName'))]",
```

Comment:
Should this be eventHubKeyResourceId or is it actually a resource? and [here](https://github.com/Azure/pcs-cli/pull/263/files#diff-030b98b55e6ff9e8234541ed87554de1R327) as well?